std/functional.d

```diff
@@ -637,6 +661,11 @@ deprecated
 
 /**
 Negates predicate `pred`.
+
+Params:
+    pred = A string or a callable
+Returns:
+    A function which calls `pred` and returns its value negated.
```

Comment:
Although a predicate by definition is a function x => bool, it might be better to say "the logical negation of its value" instead of "its value negated".